lib/actuators/include/DCMotor.h

```diff
@@ -22,26 +19,27 @@ class Motor {
    *
    * @param pwm       A PwmOut pin, driving the H-bridge enable line to control the speed
    * @param dir       A DigitalOut, set high when the motor should go forward, low when backwards
-   * @param freqInHz  Output PWM frequency, default 1kHz
+   * @param freqInHz  Output PWM frequency, default 1kHz (TODO)
```

Comment:
whats the todo for?

Reply:
There were contradicting default frequency values, not sure what it should actually be